src/controllers/raccordement/getTransmettreDateMiseEnServicePage.ts

```diff
@@ -112,11 +116,51 @@ v1Router.get(
         });
       }
 
+      const délaiCDC2022Appliqué = await ProjectEvent.findOne({
+        where: {
+          type: 'ProjectCompletionDueDateSet',
+          'payload.reason': 'délaiCdc2022',
+          projectId: projet.legacyId,
+        },
+      });
+
+      const cahierDesChargesParsé = parseCahierDesChargesRéférence(projet.cahierDesCharges);
+
+      const récupérerIntervalleDatesMeSDélaiCDC2022 = (
+        projet: CandidatureLegacyReadModel,
+        cahierDesChargesParsé: CahierDesChargesRéférenceParsed,
+      ) => {
+        const appelOffreProjet = getProjectAppelOffre({
+          appelOffreId: projet.appelOffre,
+          periodeId: projet.période,
+          familleId: projet.famille,
+        });
+
```

Comment:
pourquoi ne pas rajouter un check ici ? 
```suggestion

if (!appelOffreProjet) {
  return undefined;
}

```